public/js/game.js

```diff
@@ -35,4 +49,35 @@ socket.on('user setup', function(){
 		socket.emit('addUser');
 		console.log("kik not enabled");
 	}
-});
\ No newline at end of file
+});
+
+function renderTiles(tiles) {
+	var container = document.getElementById('tile-container');
+	var shape, bgColor;
+	for (var i = 0; i < tiles.length; i++) {
+		tile = tiles[i];
+
+		if (tile.backgroundColor === 'black') {
+			bgColor = 'red-light';
+		} else if (tile.backgroundColor === 'grey') {
+			bgColor = 'blue-light';
+		} else if (tile.backgroundColor === 'white') {
+			bgColor = 'yellow-light';
+		}
```

Comment:
lol....black maps to red-light, grey maps to blue-light, white maps to yellow-light